api/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImplTest.java

```diff
@@ -62,18 +65,24 @@
 	
 	private static final String NOT_FOUND_NAME = "not found name";
 	
-	private static final int PERSON_BIRTH_YEAR = 1999;
-	
-	private static final int WRONG_BIRTH_YEAR = 1000;
-	
 	private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd");
 	
 	private static final String BIRTH_DATE = "1999-12-20";
 	
-	private static final String NOT_FOUND_BIRTH_DATE = "1000-00-00";
+	private static final String NOT_FOUND_BIRTH_DATE = "0001-01-01";
```

Comment:
Was this change necessary?

Reply:
Yes, else the `DataRangeParam` returned an error on parsing.